integration/test_event_actions.py

```diff
@@ -124,6 +131,20 @@ def index():
                     "custom-prevent-default"
                 ).prevent_default,
             ),
+            rx.button(
+                "Throttle",
+                id="btn-throttle",
+                on_click=lambda: EventActionState.on_click_throttle.throttle(
```

Comment:
I shoud have been more specific in the test I wanted, but since lambda are used to pass pre-set parameter, when we do, do we use 
```python
lambda value: EventActionState.on_change_throttle_with_args(value, "arg1").throttle(200).stop_propagation
```
or
```python
(lambda value: EventActionState.on_change_throttle_with_args(value, "arg1")).throttle(200).stop_propagation
```
?
Though maybe it's more important to have this particular example in the docs than the tests.